modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,270 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-enabling-the-creation-of-aggregated-cluster-roles_{context}"]
+= Enabling the creation of aggregated cluster roles
+
+To enable the creation of aggregated cluster roles for the Argo CD Application Controller component of a cluster-scoped Argo CD instance, you must configure the corresponding field by editing the YAML file of the Argo CD custom resource (CR).
+
+.Procedure
+
+. In the Argo CD CR, set the value of the `.spec.aggregatedClusterRoles` field to `true`:
++
+.Example Argo CD CR
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example # <1>
+  namespace: spring-petclinic # <2>
+# ...
+spec:
+  aggregatedClusterRoles: true # <3>
+# ...
+----
+<1> The name of the cluster-scoped instance.
+<2> The namespace where you want to run the cluster-scoped instance.
+<3> The value set to `true` enables the creation of aggregated cluster roles. If you do not want to enable the creation of aggregated cluster roles, do not include this line or set the value to `false`.
++
+.Example output
+[source,terminal]
+----
+argocd.argoproj.io/example configured
+----
+
+. Verify that the `Status` field of the cluster-scoped Argo CD instance shows as `Phase: Available` by running the following command:
++
+[source,terminal]
+----
+$ oc describe argocd.argoproj.io/example -n spring-petclinic
+----
++
+.Example output
+[source,terminal]
+----
+Name:         example
+Namespace:    spring-petclinic
+Labels:       <none>
+Annotations:  <none>
+API Version:  argoproj.io/v1beta1
+Kind:         ArgoCD
+Metadata:
+  Creation Timestamp:  2024-08-14T08:20:53Z
+  Finalizers:
+    argoproj.io/finalizer
+  Generation:        3
+  Resource Version:  60437
+  UID:               57940e54-d60b-4c1a-bc4a-85c81c63ab69
+Spec:
+  Aggregated Cluster Roles:  true
```

Comment:
do we need this whole example? or could we just 
# ... 
here and then pick back up on line 102? because it's a looong scan to get down to the info we're looking for

Reply:
Addressed